.travis/set_tag.sh

```diff
@@ -3,5 +3,5 @@
 if [[ ! -z ${TRAVIS_TAG} ]]; then
     export ARCHIVE_TAG=${TRAVIS_TAG}
 else
-    DATE=$(date +%Y.%m.%d) && export ARCHIVE_TAG="nightly-$DATE"
+    DATE=$(date +%Y.%m.%d-%T) && export ARCHIVE_TAG="nightly-$DATE"
```

Comment:
While you're at it could you fix the date format to be `+%Y-%m-%dT%T`? That is  ISO8601 format (which makes it much easier to parse etc. in case someone wants to automate something with the nightlies etc.

Reply:
Sure.